.github/ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,18 @@
 <!--
 
-Thanks for filing an issue! If this is a question or feature request, just delete
-everything here and write out the request, providing as much context as you can.
+Thanks for filing an issue!
 
-If this is a usage or deployment question and not a bug report, consider asking it in one of our community supported channel. 
+If this is a usage or deployment question, and not a bug report,
+consider asking it in one of our community supported channel.
 
-Slack: https://osquery-slack.herokuapp.com/
-Stack Overflow: https://area51.stackexchange.com/proposals/117450/osquery
+StackOverflow: https://stackoverflow.com/tags/osquery
 Reddit: https://www.reddit.com/r/osquery/
 
+Slack: https://osquery-slack.herokuapp.com/
+
+If this is a feature request, just delete everything here and write
```

Comment:
Maybe,

```
+If this is a feature request, bug report, or request for comment, just delete everything here and write
+out the request, providing as much context as you can.
```

Reply:
Not sure. Most of the questions in that template are important for bug reports. If we don't want them, then we should just remove them from the template. I like adding RFC though. I'll push an update